### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (.info.form-text)

### ISSUE THREAD

**Issue by brianteeman** [joomla-projects/j4adminui#50]
#### Issue
`Elements must have sufficient color contrast` ([color-contrast](https://dequeuniversity.com/rules/axe/3.3/color-contrast?application=msftAI))
#### Target application
[Joomla Installer](http://localhost/kh/installation/index.php)
#### Element path
#installStep3 > .form-no-margin:nth-child(1) > .control-group > .info.form-text
#### Snippet
`<small class="form-text info">Select a database type form the options.</small>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 3.83 (foreground color: #7a8391, background color: #ffffff, font size: 8.4pt (11.2px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 78.0.3904.97
